docs/reference-guides/providers/howto.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 0
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Create & import a cluster using CAPI providers
+
+This guide goes over the process of creating and importing CAPI clusters with a selection of the officially supported providers.
+
+Remember that most Cluster API Providers are upstream projects maintained by the Kubernetes open-source community.
+
+## Prerequisites
+
+<Tabs>
+  <TabItem value="aws-kubeadm" label="AWS Kubeadm" default>
+    - Rancher Manager cluster with Rancher Turtles installed
+    - Cluster API Providers: you can find a guide on how to install a provider using the `CAPIProvider` resource [here](../../tasks/capi-operator/basic_cluster_api_provider_installation.md)
+        - [Infrastructure provider for AWS](https://github.com/kubernetes-sigs/cluster-api-provider-aws/)
+        - [Bootstrap/Control Plane provider for Kubeadm](https://github.com/kubernetes-sigs/cluster-api)
+    - **clusterctl** CLI - see [releases](https://github.com/kubernetes-sigs/cluster-api/releases)
```

Comment:
probably better to refer to clusterctl installation [instructions](https://cluster-api.sigs.k8s.io/user/quick-start#install-clusterctl) in the CAPI book, WDYT?